### ISSUE TITLE
undefined method `nil_class_session_path' for facebook_link helper

### ISSUE THREAD

**Issue by bitzesty** [grimen/devise_facebook_connectable#7]
using: 

```
devise (0.7.1) (ORM: mongo_mapper)
 devise_facebook_connectable (0.1.5)

class User
  devise :authenticatable, :facebook_connectable
end
```

 Looks like a re-emergence of a closed bug, looks like devise changed some mappings which were monkey patched.

**Comment by bitzesty** [comment]
invalid, forgot to specify :for, perhaps raise a warning?

**Comment by grimen** [comment]
Yes, you have to specify scope with :for. I think I should come up with a more seamless solution if only one scope is available.

**Comment by grimen** [comment]
view link helpers: Auto-detect scope, or raise error if no valid scope found or specified explicitly. Closed by ef782dce6b7d4fb20ecc5a90448b09d68a80c77e.
